runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierConfig.java

```diff
@@ -53,11 +53,22 @@ default boolean interactionIdle()
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "movementidle",
+			name = "Idle Movement Notifications",
+			description = "Configures if idle movement notifications are enabled e.g. running, walking",
+			position = 3
+	)
+	default boolean movementIdle()
+	{
+		return true;
```

Comment:
This should definitely not default to on

Reply:
I agree, I've changed this now, thanks.